### PR TITLE
Removed old BDX code

### PR DESCRIPTION
**Pull request by johantjellden**

### CODE REVIEW

File: src/main/java/com/hedvig/productPricing/service/web/InternalController.java [hunk 1]
```diff
@@ -473,23 +474,33 @@ public ProductSearchResultDTO searchPaged(ProductSearchRequestDTO searchRequest)
...
 
+  /**
+   * Remove this method as soon as the BDX tab is removed from backoffice
+   *
+   * @param year
+   * @param month
+   * @param productType
+   * @return
+   */
+  @Deprecated
```
Comment (meletios) [inline src/main/java/com/hedvig/productPricing/service/web/InternalController.java]: Why not remove it from now?

Comment (meletios) [reply src/main/java/com/hedvig/productPricing/service/web/InternalController.java]: Since nobody uses this tab is using this tab

Comment (johantjellden) [reply src/main/java/com/hedvig/productPricing/service/web/InternalController.java]: There will be an ugly error in hope.

Comment (meletios) [reply src/main/java/com/hedvig/productPricing/service/web/InternalController.java]: Correcto! Lets leave it

**Pull request merged**